content/reference/plugins/github_actions/index.md

```diff
@@ -0,0 +1,189 @@
+---
+title: "GitHub Actions Plugin"
+description: "Learn how to configure the Gatling Enterprise GitHub Actions plugin and run your simulations."
+lead: "Run your Gatling Enterprise simulations from GitHub Actions."
+date: 2022-01-04T15:00:00+00:00
+lastmod: 2022-01-04T15:00:00+00:00
+weight: 23040
+---
+
+## Purpose of this GitHub Action
+
+This Action enables you to start a Gatling Enterprise simulation directly from your GitHub Actions workflows. This plugin links a workflow with one and only one Gatling Enterprise simulation.
+
+This plugin doesn't create a new Gatling Enterprise simulation, you have to create it using the Gatling Enterprise Dashboard before, or do it using the options provided by our build tools plugins (for [Maven](https://gatling.io/docs/gatling/reference/current/extensions/maven_plugin/#working-with-gatling-enterprise-cloud), [Gradle](https://gatling.io/docs/gatling/reference/current/extensions/gradle_plugin/#working-with-gatling-enterprise-cloud), [SBT](https://gatling.io/docs/gatling/reference/current/extensions/sbt_plugin/#working-with-gatling-enterprise-cloud)).
+
+Don't forget to check out [GitHub's official documentation](https://docs.github.com/en/actions) to learn how to write CI/CD workflows with GitHub Actions.
+
+## Releases
+
+You can check out the latest releases available [from the GitHub project](https://github.com/gatling/enterprise-action/releases). You should generally only specify the major version you want to use, currently `gatling/enterprise-action@v1`.
+
+## Pre-requisites
+
+You must first [create an API token]({{< ref "../../admin/api_tokens" >}}), with at least the Start permission. It will be used by the Action to authenticate with Gatling Enterprise. We recommend storing the API Token [in a GitHub encrypted secret](https://docs.github.com/en/actions/security-guides/encrypted-secrets#using-encrypted-secrets-in-a-workflow). In the following examples, we assume the API Token is stored in a secret called `GATLING_ENTERPRISE_API_TOKEN`.
+
+We also assume that you have already [configured a simulation]({{< ref "../../user/simulations" >}}) on Gatling Enterprise. You can copy the simulation ID from the simulations list view. In the following examples, we will show the simulation ID as `xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx`.
```

Comment:
SimulationID should be syntactically correct.
UUID.zero?

`00000000-0000-0000-0000-000000000000`